tests/suites/0_stateless/05_ddl/05_0018_ddl_revoke.sql

```diff
@@ -0,0 +1,47 @@
+DROP USER IF EXISTS 'test-user';
+DROP USER IF EXISTS 'test-priv';
+DROP DATABASE IF EXISTS a;
+DROP DATABASE IF EXISTS b;
+DROP ROLE IF EXISTS 'test-role';
+DROP ROLE IF EXISTS 'test';
+DROP USER IF EXISTS 'test-user';
+DROP USER IF EXISTS 'test-priv';
+CREATE DATABASE a;
+CREATE DATABASE b;
+
+REVOKE ROLE 'test' FROM 'test-user'; -- {ErrorCode 2201}
+CREATE USER 'test-user' IDENTIFIED BY 'password';
+REVOKE ROLE 'test' FROM 'test-user';
+CREATE ROLE 'test';
+--REVOKE ROLE TEST
+GRANT SELECT ON b.* TO ROLE 'test';
+SHOW GRANTS FOR ROLE 'test';
+GRANT ROLE 'test' TO 'test-user';
+SHOW GRANTS FOR 'test-user';
+REVOKE SELECT ON a.* FROM ROLE 'test';
+SHOW GRANTS FOR ROLE 'test';
+SHOW GRANTS FOR 'test-user';
+REVOKE SELECT ON b.* FROM ROLE 'test';
+SHOW GRANTS FOR 'test-user';
+SHOW GRANTS FOR ROLE 'test';
```

Comment:
BTW, I think this test has some questions. But it no matters with this pr. So I will create a new issue to disscuss about it.

Reply:
https://github.com/datafuselabs/databend/issues/6068